torch/_dynamo/optimizations/distributed.py

```diff
@@ -41,13 +42,20 @@ def pretty_print_buckets(buckets: List[Bucket]):
         print("Please `pip install tabulate` in order to pretty-print ddp bucket sizes")
 
 
+def fx_name(fqn):
+    parts = fqn.split(".")
+    module = "self_" + "_".join(parts[:-1])
```

Comment:
Oh this is another one that is a bit fragile. Could you point to the place that defines this format?